services/movie_session.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int, cinema_hall_id: int) -> MovieSession:
+    return MovieSession.objects.create(show_time=movie_show_time,
+                                       movie_id=movie_id,
+                                       cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: str = None) -> list[MovieSession]:
+    if session_date:

```

Comment:
Return type hint here will be `QuerySet` 

Remember:`.all()`, `.filter()` returns QuerySet`
.get()`, `.delete()`, `.create()` returns model class, `-> User` for. example